sample/src/androidTest/java/com/schibsted/spain/barista/sample/introduction/IntroducingBaristaAssertions.java

```diff
@@ -62,8 +62,8 @@ public void letsIntroduceBarista() {
     assertUnchecked(R.id.unchecked_checkbox);
 
     // What's the state of the Drawer
-    assertDrawerIsOpen(R.id.drawer);
-    assertDrawerIsClosed(R.id.drawer);
+    assertDrawerIsOpen();
```

Comment:
I think we can also give the methods that check it by id or by gravity, just for the marketing.

Reply:
Does this count as a readme? Because in that case I will update it in https://github.com/SchibstedSpain/Barista/issues/151

Reply:
Well, that method is what we used to make a copy paste to update the `README.md` 👅